src/api/app/views/webui/user/_info.html.haml

```diff
@@ -38,18 +38,9 @@
                                                                  'features we develop and give us feedback on them before they go live.' } } |
       .mt-4
         - if configuration.accounts_editable?(user)
-          - if account_edit_link.present?
-            = link_to(account_edit_link, class: 'd-block') do
-              %i.fas.fa-user-edit
-              Edit your account
-          - else
-            = link_to('#', data: { toggle: 'modal', target: '#edit-modal' }, class: 'd-block') do
-              %i.fas.fa-user-edit
-              Edit your account
+          = render partial: 'webui/user/edit_account', locals: { account_edit_link: account_edit_link }
```

Comment:
I'm not sure putting those links in a partial brings benefits.

Reply:
I think there are too many `if`s here, maybe once I can refactor the action links this partial is not needed anymore...